tutorials/28_Structured_Output_With_OpenAI.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    responses_generator.run(messages=[ChatMessage.from_user(" Berlin is the capital of Germany. It has a population of 3,850,809. Paris, France's capital, has 2.161 million residents.")])
</pre><p> </p><p>remove the initial space</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/412/discussion/#comment-2581346712'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>